### PR TITLE
Add Berni's post on erasure for advertising companies

### PR DESCRIPTION
**Pull request by baltpeter**
Thanks to @berni44 for writing the post!

### CODE REVIEW

File: content/de/blog/datenloeschung-bei-werbetreibenden/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "title": "Datenlöschen bei Werbetreibenden ist nicht so einfach",
```
Comment (mal-tee) [inline content/de/blog/datenloeschung-bei-werbetreibenden/index.md:2]:
```suggestion
    "title": "Datenlöschung bei Werbetreibenden ist nicht so einfach",
```

**Review by mal-tee — DISMISSED** [review]
Great, thank you, Berni!

File: content/de/blog/datenloeschung-bei-werbetreibenden/index.md (hunk 1 above)
Comment (mal-tee) [reply content/de/blog/datenloeschung-bei-werbetreibenden/index.md:2]: Nevermind.

Comment (berni44) [reply content/de/blog/datenloeschung-bei-werbetreibenden/index.md:2]: The suggestion is fine for me.

**Pull request merged**